scripts/linters/js_ts_linter.py

```diff
@@ -436,10 +435,11 @@ def _check_angular_services_index(self) -> concurrent_task_utils.TaskResult:
 
     def _check_unknown_type(self) -> concurrent_task_utils.TaskResult:
         """Prints a list of lint errors if an unknown type is used. This lint
-        check is not enabled by default. Add proper comment if unknown is needed.
+            check is not enabled by default. Add proper comment if unknown is needed.
```

Comment:
Deindent by 4. (Follow the format of other docstrings if you are not sure.)

Reply:
Done